packages/dashboard/src/components/composed/Sidebar/Middle/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { Navbar, Badge, Indicator } from "@mantine/core";
+import { Zap, Archive, Aperture } from "react-feather";
+import NavButton from "src/components/composed/Sidebar/Middle/NavButton";
+import { useDash } from "src/hooks";
+
+function Middle(): JSX.Element {
+  const {
+    state: { providerMessages }
+  } = useDash()!;
+  const numRequests = providerMessages.size;
+  const featherIconProps = { size: 18 };
+
+  const comingSoonBadge = (
+    <Badge
+      variant="gradient"
+      gradient={{ from: "truffle-teal", to: "pink" }}
+      sx={{ opacity: 0.4 }}
+    >
+      coming soon
+    </Badge>
+  );
+
+  return (
+    <Navbar.Section grow py="sm">
+      <NavButton
+        label="Signature Requests"
```

Comment:
```suggestion
        label="Signature requests"
```

Maybe it's personal preference but I always felt like Section Headers Using All Caps was just a bit overdoing it ;)